dom0/sd-dom0-files.sls

```diff
@@ -52,6 +52,25 @@ dom0-workstation-rpm-repo:
     - require:
       - file: dom0-rpm-test-key
 
+dom0-workstation-templates-repo:
+  # Using file.blockreplace because /etc/qubes/repo-templates/ is not a .d
+  # style directory, and qvm.template_installed:fromrepo seems to only support
+  # using a repo from this file. Installing manually via a cli-command-instead?
+  file.blockreplace:
+    - name: /etc/qubes/repo-templates/qubes-templates.repo
+    - append_if_not_found: True
+    - marker_start: "### BEGIN securedrop-workstation ###"
+    - marker_end: "### END securedrop-workstation ###"
+    - content: |
+        [securedrop-workstation-templates]
+        gpgcheck=1
+        gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation
```

Comment:
I noticed that in 4.1, key files are also stored in `/etc/qubes/repo-templates/keys/` (I have duplicate `RPM-GPG-KEY-xxxx` in both locations), but this new location could be preferred to store key files with repo templates.

Reply:
Agreed :100: - I assumed the template repo key(s) were moved there and didn't want to duplicate files unnecessarily, but it looks like 4.1 has the same keys in both directories at the moment. I wonder if that's a requirement for `qvm-template` to work.

Reply:
nope, currently it's not (from my limited testing anyway)- it works with keys in the /etc/pki directory, as long as that's specified in the .repo file, although long-term I imagine that may change.